src/components/Avatar/Avatar.js

```diff
@@ -75,7 +76,7 @@ class Avatar extends Component {
           height={iconSize}
           width={iconSize}
           description={description}
-          name={'profiles--glyph'}
+          name={defaultIconName}
```

Comment:
Can we go with `iconName` here just for naming consistency with the rest of the icon variables.

Reply:
i prefer `name` to `icon` (because thats what the Icon prop is called) but i think thats a bit too generic, i wanted the concept of `default` in there somewhere to represent that this is only relevant if the imgurl doesn't work

Reply:
Makes sense 👍